packages/account/src/Components/poa/common-mistakes/__tests__/common-mistake-example.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CommonMistakeExamples from '../common-mistake-examples';
+
+jest.mock('Assets/ic-error-badge.svg', () => jest.fn(() => 'ErrorIcon'));
+
+describe('<CommonMistakeExamples/>', () => {
```

Comment:
No need to have `describe` if there is only a single test.

Reply:
describe is for making a block for tests, as per our documentation, we should use it to wrap our tests.
let's follow our common approaches

Reply:
@yauheni-deriv that is true and since there is only a single test then we do not need to create a block of tests.
Are you planning to add more tests later on? 

ref: https://jestjs.io/docs/api#describename-fn

Reply:
I agree, that we can skip describe. I just try to follow our common practice in the code. 
we do have couple of places with one test case, but wrapped with describe. So I'd suggest to keep our common practice.
cc: examples: CFDAccountCopy, CFDPOI, LoadingCFDRealAccountDisplay, LoginButton etc...